nimare/__init__.py

```diff
@@ -7,9 +7,6 @@
     warnings.simplefilter("ignore")
     from .dataset import Dataset
     from .meta import cbma
```

Comment:
No, but this way we can access `meta` without importing it separately. So `import nimare; nimare.meta.cbma.ALE()` works, instead of just `from nimare.meta import cbma; cbma.ALE()`.

Reply:
See above.